dotCMS/src/main/webapp/html/portlet/ext/useradmin/view_users_js_inc.jsp

```diff
@@ -386,7 +387,13 @@
 
 		var userEmail;
 		if(emailChanged){
-			var pattern=/^([a-zA-Z0-9_.-])+@([a-zA-Z0-9_.-])+\.([a-zA-Z])+([a-zA-Z])+/;
+            <%
+                 String emailRegex = Config.getStringProperty(com.dotmarketing.util.WebKeys.DOTCMS_USE_REGEX_TO_VALIDATE_EMAILS, null);
```

Comment:
1. Yes @wezell , it is. Do you thing it can affect security in the app (xss attacks), because this regex allows single quotes? Otherwise, I agree to make it our default 
2. Good suggestion, I've committed the improvement.